client/blocks/conversation-caterpillar/docs/fixtures.js

```diff
@@ -52,4 +52,95 @@ export const comments = [
 		},
 		date: '2016-04-18T15:23:00+00:00',
 	},
```

Comment:
i'd definitely say we are at a point where it'd be better to find real posts.

that way we can easily get multiple different kinds of examples from wordpress en.blog